analyses/independent-samples/02-generate-indpependent-rnaseq.R

```diff
@@ -0,0 +1,100 @@
+# 02-generate-independent-rnaseq.R
+# Krutika Gaonkar for D3b
+#
+# Purpose: Generate tables of independent rna-seq specimens 
+# Option descriptions
+# -f, --histology_file : File path to where you would like the annotation_rds file to be
+#               stored
+# -o,--output_directory : Output directory
+# example invocation:
+# Rscript analyses/independent-samples/02-generate-independent-rnaseq.R \
+#   -f data/pbta-histologies.tsv \
+#   -o analyses/independent-samples/results
```

Comment:
Can you add this as a step in `run-independent-samples.sh` 

Since this is already a step in CI, this will allow us to make sure that things are all running as they should.

Reply:
will do!